apps/src/javalab/JavabuilderConnection.js

```diff
@@ -349,15 +350,19 @@ export default class JavabuilderConnection {
 
   onUnsupportedNeighborhoodMessage() {
     if (!this.seenUnsupportedNeighborhoodMessage) {
-      this.onOutputMessage(javalabMsg.unsupportedNeighborhoodMessage());
+      this.onOutputMessage(
+        '[WARNING] ' + getUnsupportedMiniAppMessage(CsaViewMode.NEIGHBORHOOD)
```

Comment:
should we just use `[EXCEPTION]` here to be consistent?

Reply:
Yeah, that's fine with me. It technically wouldn't be an exception that caused it but I don't think it makes a huge difference to the user